src/test/resources/data/StudentFeedbackQuestionSubmitPageUiTest.json

```diff
@@ -29,6 +29,7 @@
             "name":"Teammates Test",
             "email":"SFQSubmitUiT.instr@gmail.tmt",
             "role":"Co-owner",
+            "isDisplayedToStudents":true,
```

Comment:
I think there is something strange happening. On master branch, `StudentFeedbackQuestionSubmitPageUiTest` passes (suggesting that hidden instructors are shown), but manual testing shows that hidden instructors are not shown when submitting feedback as a student.